### ISSUE TITLE
Print the number of directory items in --long mode.

### ISSUE THREAD

**Issue by notramo** [ogham/exa#336]
It currently prints a `-` in the `Size` column for directories when using `--long` mode.
Printing the number of child entries would be better.

**Comment by dieggsy** [comment]
I think that would be pretty misleading as that's not a size. The number could be (probably rightfully) miscontsrued as file size if in the same column.

**Comment by notramo** [comment]
You're right. It can be ambigous since file sizes lower than 1KiB are displayed as bare numbers. What about appending "items":
```
Permissions  Size    User  Date Modified  Git Name
drwxr-xr-x   1 items user1 13 dec   11:59  -- lib
drwxr-xr-x   3 items user1 13 dec   11:59  -- spec
drwxr-xr-x   2 items user1 13 dec   13:12  -M src
.rw-r--r--       755 user1 13 dec   11:59  -- README.md
.rw-r--r--       226 user1 13 dec   11:59  -- shard.yml
```

Other solution: display the number in a different color instead of the `items` prefix.

**Comment by lilyball** [comment]
`ls -l` prints number of direntries. I think we should just ahead and print that; there are other ways (colors, classification symbols) you can use to disambiguate directories from files.

I don't like appending "items" because that expands the column size. Coloring it separately might be nice though (perhaps use the configured directory color; not whatever the filename is, in case that's overridden by more specific colors, just the generic color you'd get for a directory that isn't overridden by anything else).

**Comment by mroutis** [comment]
@lilyball, the column next to the _permissions_ is the _number hardlinks_ associated to that entry, you could already do that with `--links`
